docs/topics/proc_accessing-jolokia-using-hawtio-on-openshift.adoc

```diff
@@ -0,0 +1,32 @@
+[#accessing-jvm-metrics-using-jolokia]
+= Accessing JVM metrics using Jolokia on OpenShift
+
+link:https://jolokia.org/documentation.html[Jolokia^] is a built-in lightweight solution for accessing JMX (Java Management Extension) metrics over HTTP on OpenShift.
+It allows you to access CPU, storage, and memory usage data collected by JMX over an HTTP bridge using a REST interface and JSON-formatted message payloads, making it suitable for monitoring cloud applications, due to its comparably high speed and low resource-requirements.
+
+
+All Fabric8 container images expose port `8778` by default on the host of your {OpenShiftLocal}.
```

Comment:
How are Fabric8 images relevant in this context?

Reply:
@Ladicek I see, they aren't really:) I will remove the references to Fabric8 images form this section.